.github/workflows/release.yml

```diff
@@ -45,11 +46,35 @@ jobs:
         uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # v4.0.0
         with:
           go-version: ${{ needs.get-go-version.outputs.go-version }}
+      - name: Login to docker
+        uses: docker/login-action@v2.1.0
+        with:
+          username: cipipelinetessell
+          password: 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
```

Comment:
I think you leaked your CI password on Docker here, you should change this password immediately.

Reply:
@ayushtessell I saw you doing some commits on the fork that this PR comes from and just wanted to make sure someone saw this leaked GitHub token and revoked it

Reply:
Thanks @JenGoldstrich ; We were debugging something and didn't realise its public repo. We revoked the token.